e2ev2/tests/user-create.test.js

```diff
@@ -62,8 +61,6 @@ browserTest('create user', async browser => {
   await page.keyboard.press('Tab')
   await page.keyboard.type('12345')
   await page.keyboard.press('Tab')
-  await page.keyboard.type('United')
```

Comment:
United States is selected by default, so you do not have to change the dropdown.